modules/nodes-pods-using-example.adoc

```diff
@@ -18,103 +18,52 @@ kind: Pod
-  qosClass: BestEffort
+          drop: ["ALL"]
+      resources:
+        limits:
+          memory: "100Mi"
+          cpu: "1"
+        requests:
+          memory: "100Mi"
+          cpu: "1"
+  volumes: <8>
+  - name: cache-volume
+    emptyDir:
+      sizeLimit: 500Mi
 ----
 
 <1> Pods can be "tagged" with one or more labels, which can then be used to select and manage groups of pods in a single operation. The labels are stored in key/value format in the `metadata` hash.
 <2> The pod restart policy with possible values `Always`, `OnFailure`, and `Never`. The default value is `Always`.
 <3> {product-title} defines a security context for containers which specifies whether they are allowed to run as privileged containers, run as a user of their choice, and more. The default context is very restrictive but administrators can modify this as needed.
 <4> `containers` specifies an array of one or more container definitions.
-<5> The container specifies where external storage volumes are mounted within the container. In this case, there is a volume for storing access to credentials the registry needs for making requests against the {product-title} API.
+<5> The container specifies where external storage volumes are mounted within the container.
 <6> Specify the volumes to provide for the pod. Volumes mount at the specified path. Do not mount to the container root, `/`, or any path that is the same in the host and the container. This can corrupt your host system if the container is sufficiently privileged, such as the host `/dev/pts` files. It is safe to mount the host by using `/host`.
 <7> Each container in the pod is instantiated from its own container image.
-<8> Pods making requests against the {product-title} API is a common enough pattern that there is a `serviceAccount` field for specifying which service account user the pod should authenticate as when making the requests. This enables fine-grained access control for custom infrastructure components.
-<9> The pod defines storage volumes that are available to its container(s) to use. In this case, it provides an ephemeral volume for a `secret` volume containing the default service account tokens. 
+<8> The pod defines storage volumes that are available to its container(s) to use.
 +
-If you attach persistent volumes that have high file counts to pods, those pods can fail or can take a long time to start. For
-more information, see link:https://access.redhat.com/solutions/6221251[When using Persistent Volumes with high file counts in OpenShift, why do pods fail to start or take an excessive amount of time to achieve "Ready" state?].
+If you attach persistent volumes that have high file counts to pods, those pods can fail or can take a long time to start. For more information, see link:https://access.redhat.com/solutions/6221251[When using Persistent Volumes with high file counts in OpenShift, why do pods fail to start or take an excessive amount of time to achieve "Ready" state?].
```

Comment:
Doesn't seem like you wrote this line yourself, but just an FYI that this link takes you to an article with a different title:
```suggestion
If you attach persistent volumes that have high file counts to pods, those pods can fail or can take a long time to start. For more information, see link:https://access.redhat.com/solutions/6221251[Pods using Persistent Volumes with high file counts fail to start or take an excessive amount of time in OpenShift].
```